angr/analyses/cfg/cfg_accurate.py

```diff
@@ -1642,7 +1642,8 @@ def _handle_successor(self, job, successor, successors):
 
         # Fix target_addr for syscalls
         if suc_jumpkind.startswith("Ijk_Sys"):
-            target_addr = self.project.simos.syscall(new_state).addr
+            if self.project.simos.syscall(new_state):
```

Comment:
I would make it explicit here (checking the syscall is not `None`), and then save the result to the `simos.syscall(new_state)` and reuse it in the next line.